lib/tasks/build.js

```diff
@@ -3,13 +3,12 @@
 const path = require('path');
 const merge = require('merge-stream');
 const BowerPlugin = require('bower-webpack-plugin');
-const webpack = require('webpack-stream');
+const webpack = require('webpack');
+const webpackStream = require('webpack-stream');
```

Comment:
actually no it wasn't. so the `require('webpack')` should've errored. Are prod builds covered by tests?